admin_site/templates/system/image_versions.html

```diff
@@ -0,0 +1,83 @@
+{% extends "site_with_navigation.html" %}
+
+{% block specific_title %}
+Versioner
+{% endblock %}
+
+{% block javascripts %}
+
+{% endblock %}
+
+{% block image_versions_active %}
+class="active"
+{% endblock %}
+
+{% block specific_content %}
+
+<ul class="sublevelnav">
+  {% if site_allowed %}
+    {% for maj_v in major_versions %}
+    <li {% if request.resolver_match.url_name == '/site/{{ site.url }}/image-versions/{{ maj_v }}' %}class="active"{% endif %}>
```

Comment:
This is the reason why everything looks selected -- you unconditionally put `class="active"` in every element.

You might want to ~steal~borrow this implementation from `site_groups.html`:

```
      {% for group in site.groups.all %}
        <li {% ifequal group selected_group %}class="active"{% endifequal %}>```

Reply:
It has been done. I had assign some stuff for the context object in order to highlight the targeted link. I'll push the changes later today.